src/soundRecorder.js

```diff
@@ -85,7 +85,10 @@ define(function (require) {
 
     this._workletNode = new AudioWorkletNode(ac, processorNames.recorderProcessor, {
       outputChannelCount: [this._outputChannels],
-      processorOptions: { numInputChannels: this._inputChannels }
+      processorOptions: {
+        numInputChannels: this._inputChannels,
+        bufferSize: 1024
```

Comment:
if we set this to 4096, it matches the buffer size of the ScriptProcessor in Firefox. Otherwise, we get choppy recordings